packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -29,39 +54,37 @@ const SelectablePillRoot = styled.button<SelectablePillProps>((props) => {
     })
   }
 
-  return {
+  const css: CSSObject = {
     display: 'inline-flex',
     alignItems: 'center',
     justifyContent: 'center',
     gap: theme.spacing.xs,
-    height: theme.spacing.unit6,
-    backgroundColor: theme.color.special.white,
-    paddingInline: theme.spacing.unit3,
-    color: theme.color.text.subdued,
+    color: theme.color.text.default,
+    backgroundColor: theme.color.background.white,
     cursor: 'pointer',
     userSelect: 'none',
     transition: 'all 0.12s ease-out',
+    textWrap: 'nowrap',
     border: `1px solid ${theme.color.border.strong}`,
     borderRadius: theme.cornerRadius['2xl'],
-
+    ...(size === 'small' && {
+      height: theme.spacing.unit6,
+      paddingInline: theme.spacing.unit3
```

Comment:
`theme.spacing.m`?

I think the height is fine as units, but padding we should use the spacing presets

Reply:
yeah lets use .m